src/extensions/default/AutoUpdate/main.js

```diff
@@ -624,6 +626,15 @@ define(function (require, exports, module) {
         if (checkIfOnline()) {
             postMessageToNode(MessageIds.PERFORM_CLEANUP, ['.json'], true);
         } else {
+            enableCheckForUpdateEntry(true);
+            UpdateStatus.cleanUpdateStatus();
+            HealthLogger.sendAnalyticsData(
```

Comment:
I also see analytics related stuff here. Shouldn't this be in a separate PR??

Reply:
No, Here It is just a call to api to send Analytics Data. No Framework level change.